contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -23,7 +23,7 @@ const SYNC_BUFFER = 1000;
 
 // When this many seconds pass without a syncable
 // change being registered, sync changes!
-const SYNC_IF_NO_CHANGES_FOR = 2;
+const SYNC_IF_NO_CHANGES_FOR = 20;
```

Comment:
Temporary change for the purpose of testing that will be reverted

Reply:
This has not been reverted yet. Best to just maintain these kinds of changes in local state, rather than committing.

Reply:
That was for testing done by others though I wasn't able to build a demo server so I will revert now and can add this tip to comments